### DISCUSSION TITLE
How to mount content script to a dynamic element?

### DISCUSSION THREAD

**Discussion by bnn1** [wxt-dev/wxt#292] [Q&A] [answered]
Hi guys. I have a web page that has dynamic content in it. On the initial page load the element I want to mount my content script to is not in the page yet.
Does WXT have a mechanism that allows mounting content script to elements when they appear on the page? or do I need to setup the mutation observer and handle all that stuff myself?

**Comment by aklinker1** [comment, marked as answer]
You'll need to do it yourself right now.

Eventually, I plan on adding this, but I want to focus on releasing v1.0.0 and this will come after that.

**Reply by bnn1** [reply]
I see. WXT is a great framework anyway. Thank you for your answer
